inference-engine/tests/functional/plugin/cpu/single_layer_tests/depth_to_space.cpp

```diff
@@ -85,24 +135,22 @@ const auto cpuParams_ndhwc_sse42 = CPUSpecificParams {{ndhwc}, {ndhwc}, {"jit_ss
 const auto cpuParams_nhwc_ref = CPUSpecificParams {{nhwc}, {nhwc}, {"ref_any"}, {"ref_any"}};
 const auto cpuParams_ndhwc_ref = CPUSpecificParams {{ndhwc}, {ndhwc}, {"ref_any"}, {"ref_any"}};
 
-
-const std::vector<InferenceEngine::Precision> inputPrecisions = {
-        InferenceEngine::Precision::FP32,
-        InferenceEngine::Precision::BF16,
-        InferenceEngine::Precision::I8
-};
-
-const std::vector<DepthToSpace::DepthToSpaceMode> depthToSpaceModes = {
-        DepthToSpace::DepthToSpaceMode::BLOCKS_FIRST,
-        DepthToSpace::DepthToSpaceMode::DEPTH_FIRST
-};
-
-const std::vector<std::vector<size_t >> inputShapesBS2_4D = {
-        {1, 64, 1, 1}, {1, 64, 1, 3}, {1, 128, 3, 3}, {2, 128, 1, 1}, {1, 192, 2, 2}, {2, 256, 2, 3}, {1, 512, 2, 1}
+const std::vector<DepthToSpaceInputShapes> inputShapesBS2_4D = {
+        {{}, {{1, 64,  1, 1}}},
```

Comment:
You can use `ov::test::static_shapes_to_test_representation` again please refer to Softmax SL common test instance.